docs/pages/management/guides/joining-services-kubernetes-serviceaccount.mdx

```diff
@@ -0,0 +1,267 @@
+$ kubectl get deployments -n teleport
+NAME             READY   UP-TO-DATE   AVAILABLE   AGE
+teleport-auth    2/2     2            2           6d20h
+teleport-proxy   2/2     2            2           6d20h
+
+# Here, the deployment name is "teleport-auth".
+```
+
+Then run the following command to execute the `tctl create` command from inside
+one of the Auth Service pods:
+
+```code
+$ kubectl exec -i -n teleport deployment/teleport-auth -- tctl create < token.yaml
+```
+
+Finally, validate the token was successfully created:
+
+```code
+$ kubectl exec -i -n teleport deployment/teleport-auth tctl get token/kubernetes-token
+
+kind: token
+metadata:
+  expires: "3000-01-01T00:00:00Z"
+  name: kubernetes-token
+spec:
+  join_method: kubernetes
+  roles:
+  - App
+version: v2
+```
+
+</TabItem>
+</Tabs>
+
+## Step 2/5. Deploy a demonstration HTTP app
+
+In this step, we deploy a demonstration HTTP application and don't expose it
+publicly. Instead, we will manage access to this application with the Teleport
+Application Service, which we will register with Teleport using the Kubernetes
+join method.
+
+```code
+$ kubectl create namespace demo-app
+namespace/demo-app created
+
+$ kubectl create deployment --image=nginx --namespace demo-app --port=80 demo-app
+deployment.apps/demo-app created
+
+$ kubectl expose deployment demo-app -n demo-app --port=80 --target-port=80 --selector='app=demo-app'
+service/demo-app exposed
+```
+
+Validate the application pods are running and ready with the following command:
+
+```code
+$ kubectl get pods -n demo-app
+NAME                        READY   STATUS    RESTARTS   AGE
+demo-app-7664d59cb8-bvbmz   1/1     Running   0          67s
+```
+
+## Step 3/5. Configure your Application Access Service
+
+
+Configure the `teleport-kube-agent` chart to deploy Teleport instances running
+the Application Access Service by creating a `values.yaml` file with the
+following content:
+
+```yaml
+# values.yaml
+
+# Public address of the Teleport cluster with port.
+# You must replace the placeholder with your proxy address.
+proxyAddr: "teleport.example.com:443"
+
+# Comma-separated list of services the `teleport-kube-agent` chart must run
+# (supported values are: kube,db,app)
+# In this guide we only deploy app access.
+# Adding more services here also requires to add role to the provision token created in step 1.
+roles: app
+
+joinParams:
+  method: "kubernetes"
+  # this must match the provision token created in Step 1.
+  tokenName: "kubernetes-token"
+  
+apps:
+  - name: demo-app
+    uri: "http://demo-app.demo-app.svc.cluster.local:80"
+```
+
+## Step 4/5. Deploy the Application Access Service
+
+
+To use the token created in Step 1, the joining instance must run in the same
+Kubernetes cluster as the Auth Service and have a Kubernetes ServiceAccount
+token mounted.
+[The `teleport-kube-agent` chart](../../reference/helm-reference/teleport-kube-agent.mdx)
+will take care of this by default.
```

Comment:
```suggestion
The `teleport-kube-agent` chart that you will install in this section will take care of this by default.
```
I don't think we need to link the user to another docs page since we are referring to this chart in the current step.